hangman/lib/hangman/game_listener.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Hangman.GameListener do
```

Comment:
A couple of people did something like this.

Why not just use the fact that new_game returns a pid, and the client can pass that pid into Hangman.make_move and .tally, and the beam will work out where to send it to?

Reply:
I guess I'm not fully understanding what you are suggesting.  Could you give a code example?

Reply:
Each HumanPlayer client could look up the node running the game, then find the supervisor process by name. It then does something like this:

~~~ elixir
  defp new_game() do
    Node.connect(@hangman_server)
    :rpc.call(@hangman_server,
      Hangman,
      :new_game,
      [])
  end
~~~

This returns the pid of the game server on that node, and you ca then just call `Hangman.make_move(pid, "x")`